docker/lwua-dereferencer/dereferencer-py/dereferencer/dereference.py

```diff
@@ -1,128 +1,70 @@
 import logging
 import os
-import yaml
-from rdflib import Graph
-from datetime import datetime, timedelta
-from pathlib import Path
+from pytravharv import TargetStore, TravHarvConfigBuilder, TravHarvExecuter
 from .helpers import resolve_path
-from .graphdb import uri_list, get_registry_of_lastmod, writeStoreToGraphDB
-from .derefEntity import DerefUriEntity
+from pathlib import Path
+import time
 
 log = logging.getLogger(__name__)
 
+GDB_URL = "http://graphdb:7200/repositories/lwua23"
+
 
 def config_path_from_config():
-    local_default = str(resolve_path("./configs", versus="dotenv"))
+    local_default = str(resolve_path("./config", versus="dotenv"))
     folder_name = os.getenv("CONFIG_FILES_FOLDER", local_default)
     test = "/config"
     return Path(test).absolute()
 
 
-def isExpired(lastmod: datetime, cache_lifetime: int):
-    # check if the cache_lifetime is set
-    if cache_lifetime == 0:
-        return True
-    # check if the lastmod is set
-    if lastmod is None:
-        return True
-    # check if the lastmod is older than the cache_lifetime
-    if lastmod + timedelta(minutes=cache_lifetime) < datetime.now():
-        return True
-    # if the lastmod is not older than the cache_lifetime, return false
-    return False
-
-
 class Dereference:
     def __init__(self):
         pass
 
+    def init_targetstore(self, url):
+        try:
+            return TargetStore.TargetStore(url)
+        except Exception as e:
+            log.error("init targetstore failed, trying again in 1s")
+            log.error("error: {e}")
+            time.sleep(1)
+            self.init_targetstore(url)
+
     def run_dereference(self):
         log.info("running dereference")
 
         config_folder_path = config_path_from_config()
-        log.info(f"run_dereference on config files in {config_folder_path}")
-        # get all the config files in the config folder
-        # the files should be in yml or yaml format and should start with
-        # dereference
-        config_files = [f for f in config_folder_path.glob("dereference*.yml")]
+        log.info(f"run_dereference on config files in {str(config_folder_path)}")
+
+        config_files = [f for f in config_folder_path.glob("*.yml")]
```

Comment:
seems unnecessary to make a list from a generator object

Reply:
Sounds like a remark for the pyTravHarv module itself.